static/js/popover_menus.js

```diff
@@ -87,4 +107,196 @@ export function initialize() {
             compose_mobile_button_popover_displayed = false;
         },
     });
+
+    // Message actions popover.
+    delegate("body", {
+        target: ".actions_hover i.zulip-icon-ellipsis-v-solid",
+        placement: "bottom",
+        onShow(instance) {
```

Comment:
Should this appear on a single tap, rather than a long-press, in responsive mobile design?  That feels like what I'd expect to happen, at least.